modules/sdk-core/src/bitgo/lightning/lightning.ts

```diff
@@ -114,4 +118,32 @@ export class Lightning implements ILightning {
       throw new Error(`error(s) parsing response body: ${errors}`);
     });
   }
+
+  public async decodeLnurlPay(lnurl: string): Promise<LnurlPayResponse & { domain: string }> {
+    const url = decodeLnurl(lnurl);
+    const body = await this.bitgo.get(url).result();
+    const decodedRes = decodeOrElse(LnurlPayResponse.name, LnurlPayResponse, body, (errors) => {
+      throw new Error(`error(s) parsing lnurl response: ${errors}`);
+    });
+    const { availableBalance } = await this.getBalance();
+    const domain = new URL(url).hostname;
+
+    decodedRes.maxSendable = Math.min(decodedRes.maxSendable / 1000, availableBalance);
```

Comment:
We should round this down, if we something like 3001 millisats we don't want to have `3.001` but rather `3` in the interest of continuing to work with whole satoshi units. So make this a `Math.floor` (and get rid of `availableBalance` per my other comment).